diff --git a/src/qlever/Qleverfiles/Qleverfile.dblp b/src/qlever/Qleverfiles/Qleverfile.dblp
--- a/src/qlever/Qleverfiles/Qleverfile.dblp
+++ b/src/qlever/Qleverfiles/Qleverfile.dblp
@@ -29,7 +29,7 @@ CACHE_MAX_SIZE     = 5G
 TIMEOUT            = 300s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.dblp-plus b/src/qlever/Qleverfiles/Qleverfile.dblp-plus
--- a/src/qlever/Qleverfiles/Qleverfile.dblp-plus
+++ b/src/qlever/Qleverfiles/Qleverfile.dblp-plus
@@ -26,7 +26,7 @@ CACHE_MAX_SIZE              = 10G
 CACHE_MAX_SIZE_SINGLE_ENTRY = 2G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.dbpedia b/src/qlever/Qleverfiles/Qleverfile.dbpedia
--- a/src/qlever/Qleverfiles/Qleverfile.dbpedia
+++ b/src/qlever/Qleverfiles/Qleverfile.dbpedia
@@ -23,7 +23,7 @@ MEMORY_FOR_QUERIES = 10G
 CACHE_MAX_SIZE     = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.default b/src/qlever/Qleverfiles/Qleverfile.default
--- a/src/qlever/Qleverfiles/Qleverfile.default
+++ b/src/qlever/Qleverfiles/Qleverfile.default
@@ -38,7 +38,7 @@ ACCESS_TOKEN =
 # will be downloaded automatically. Use SYSTEM = native to use self-compiled
 # binaries `qlever-
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 # UI_PORT specifies the port of the QLever UI web app, when you run `qlever ui`.
diff --git a/src/qlever/Qleverfiles/Qleverfile.dnb b/src/qlever/Qleverfiles/Qleverfile.dnb
--- a/src/qlever/Qleverfiles/Qleverfile.dnb
+++ b/src/qlever/Qleverfiles/Qleverfile.dnb
@@ -33,7 +33,7 @@ MEMORY_FOR_QUERIES = 5G
 CACHE_MAX_SIZE     = 2G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.fbeasy b/src/qlever/Qleverfiles/Qleverfile.fbeasy
--- a/src/qlever/Qleverfiles/Qleverfile.fbeasy
+++ b/src/qlever/Qleverfiles/Qleverfile.fbeasy
@@ -22,7 +22,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.freebase b/src/qlever/Qleverfiles/Qleverfile.freebase
--- a/src/qlever/Qleverfiles/Qleverfile.freebase
+++ b/src/qlever/Qleverfiles/Qleverfile.freebase
@@ -21,7 +21,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 10G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.imdb b/src/qlever/Qleverfiles/Qleverfile.imdb
--- a/src/qlever/Qleverfiles/Qleverfile.imdb
+++ b/src/qlever/Qleverfiles/Qleverfile.imdb
@@ -29,7 +29,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.ohm-planet b/src/qlever/Qleverfiles/Qleverfile.ohm-planet
--- a/src/qlever/Qleverfiles/Qleverfile.ohm-planet
+++ b/src/qlever/Qleverfiles/Qleverfile.ohm-planet
@@ -34,7 +34,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 4G
 WARMUP_CMD                  = curl -s https://qlever.cs.uni-freiburg.de/petrimaps/query --data-urlencode "query=PREFIX geo: <http://www.opengis.net/ont/geosparql#> PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> PREFIX osm: <https://www.openstreetmap.org/> SELECT ?osm_id ?geometry WHERE { ?osm_id geo:hasGeometry/geo:asWKT ?geometry . ?osm_id rdf:type osm:node } LIMIT 1" --data-urlencode "backend=https://qlever.cs.uni-freiburg.de/api/${data:NAME}" > /dev/null
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.olympics b/src/qlever/Qleverfiles/Qleverfile.olympics
--- a/src/qlever/Qleverfiles/Qleverfile.olympics
+++ b/src/qlever/Qleverfiles/Qleverfile.olympics
@@ -24,7 +24,7 @@ CACHE_MAX_SIZE     = 2G
 TIMEOUT            = 30s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.orkg b/src/qlever/Qleverfiles/Qleverfile.orkg
--- a/src/qlever/Qleverfiles/Qleverfile.orkg
+++ b/src/qlever/Qleverfiles/Qleverfile.orkg
@@ -23,7 +23,7 @@ MEMORY_FOR_QUERIES = 10G
 CACHE_MAX_SIZE     = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.osm-country b/src/qlever/Qleverfiles/Qleverfile.osm-country
--- a/src/qlever/Qleverfiles/Qleverfile.osm-country
+++ b/src/qlever/Qleverfiles/Qleverfile.osm-country
@@ -31,7 +31,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 5G
 TIMEOUT                     = 100s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.osm-planet b/src/qlever/Qleverfiles/Qleverfile.osm-planet
--- a/src/qlever/Qleverfiles/Qleverfile.osm-planet
+++ b/src/qlever/Qleverfiles/Qleverfile.osm-planet
@@ -32,7 +32,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 20G
 TIMEOUT                     = 600s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.osm-planet-from-pbf b/src/qlever/Qleverfiles/Qleverfile.osm-planet-from-pbf
--- a/src/qlever/Qleverfiles/Qleverfile.osm-planet-from-pbf
+++ b/src/qlever/Qleverfiles/Qleverfile.osm-planet-from-pbf
@@ -35,7 +35,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 10G
 TIMEOUT                     = 600s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.pubchem b/src/qlever/Qleverfiles/Qleverfile.pubchem
--- a/src/qlever/Qleverfiles/Qleverfile.pubchem
+++ b/src/qlever/Qleverfiles/Qleverfile.pubchem
@@ -124,7 +124,7 @@ MEMORY_FOR_QUERIES = 20G
 TIMEOUT            = 600s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.scientists b/src/qlever/Qleverfiles/Qleverfile.scientists
--- a/src/qlever/Qleverfiles/Qleverfile.scientists
+++ b/src/qlever/Qleverfiles/Qleverfile.scientists
@@ -22,7 +22,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.uniprot b/src/qlever/Qleverfiles/Qleverfile.uniprot
--- a/src/qlever/Qleverfiles/Qleverfile.uniprot
+++ b/src/qlever/Qleverfiles/Qleverfile.uniprot
@@ -67,7 +67,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 5G
 TIMEOUT	                    = 300s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.vvz b/src/qlever/Qleverfiles/Qleverfile.vvz
--- a/src/qlever/Qleverfiles/Qleverfile.vvz
+++ b/src/qlever/Qleverfiles/Qleverfile.vvz
@@ -24,7 +24,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 10G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.wikidata b/src/qlever/Qleverfiles/Qleverfile.wikidata
--- a/src/qlever/Qleverfiles/Qleverfile.wikidata
+++ b/src/qlever/Qleverfiles/Qleverfile.wikidata
@@ -35,7 +35,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 5G
 TIMEOUT                     = 600s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = adfreiburg/qlever
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.wikidata-munged b/src/qlever/Qleverfiles/Qleverfile.wikidata-munged
--- a/src/qlever/Qleverfiles/Qleverfile.wikidata-munged
+++ b/src/qlever/Qleverfiles/Qleverfile.wikidata-munged
@@ -40,7 +40,7 @@ CACHE_MAX_SIZE_SINGLE_ENTRY = 5G
 TIMEOUT                     = 600s
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.wikipathways b/src/qlever/Qleverfiles/Qleverfile.wikipathways
--- a/src/qlever/Qleverfiles/Qleverfile.wikipathways
+++ b/src/qlever/Qleverfiles/Qleverfile.wikipathways
@@ -33,7 +33,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 5G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
diff --git a/src/qlever/Qleverfiles/Qleverfile.yago-4 b/src/qlever/Qleverfiles/Qleverfile.yago-4
--- a/src/qlever/Qleverfiles/Qleverfile.yago-4
+++ b/src/qlever/Qleverfiles/Qleverfile.yago-4
@@ -26,7 +26,7 @@ ACCESS_TOKEN       = ${data:NAME}
 MEMORY_FOR_QUERIES = 30G
 
 [runtime]
-SYSTEM = docker
+SYSTEM = native
 IMAGE  = docker.io/adfreiburg/qlever:latest
 
 [ui]
